client/src/app/shared/models/event-management/meeting.ts

```diff
@@ -20,10 +22,14 @@ export type AgendaItemCreation = 'always' | 'never' | 'default_yes' | 'default_n
  */
 export type BallotPaperSelection = 'NUMBER_OF_DELEGATES' | 'NUMBER_OF_ALL_PARTICIPANTS' | 'CUSTOM_NUMBER';
 
-export interface Settings {
+const MEETING_NAME = 'meeting';
+export class Settings {
```

Comment:
As I mentioned before, only classes and class-members can have decorators. Therefore, I had to "convert" the interface `Settings` into a class.